### DISCUSSION TITLE
Help with Understanding Webhooks

### DISCUSSION THREAD

**Discussion by Redlyf85** [community/community#47460] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I was asked to provide a webhook and I really don’t think I know what to do

**Comment by ghostinhershell** [comment]
Hi @Redlyf85! 👋 

Welcome to GitHub and the GitHub Community 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Comment by ghostinhershell** [comment]
> I was asked to provide a webhook and I really don’t think I know what to do

Can you clarify your question a bit more? Are you looking to learn more about webhooks or were you asked to create or provide a specific webhook? Giving as much information as you can will help others jump in and answer your question.

Here are a couple of articles that may help point you in the right direction:
- [About webhooks](https://docs.github.com/en/webhooks-and-events/webhooks/about-webhooks)
- [Creating webhooks](https://docs.github.com/en/webhooks-and-events/webhooks/creating-webhooks)

Cheers!
